docs/content/concepts.mdx

```diff
@@ -651,10 +651,10 @@ For more information, see [Modeling Public Access](./modeling/public-access.mdx)
       id: './authorization-concepts',
     },
     {
-      title: 'Configuration Language',
-      description: 'Learning about the FGA configuration language',
-      link: './configuration-language',
-      id: './configuration-language',
+      title: 'Modeling Language',
+      description: 'Learning about the FGA modeling language',
```

Comment:
In docs/content/getting-started/immutable-models.mdx we use `{ProductName}`, here we use `FGA` - which do you prefer?

Reply:
I think that Andres strongly prefers {ProductName} because it means the content can be recycled.